src/com/pentalog/pentastagiu/homework4/functionalInterface/Application.java

```diff
@@ -0,0 +1,26 @@
+package com.pentalog.pentastagiu.homework4.functionalInterface;
+
+
+
+public class Application {
+    public static void main(String[] args) {
+
+
+   Application exampleFunction= new Application();
```

Comment:
Puteai sa faci metoda operate statica pentru a nu mia fi nevoie sa iti instantiezi clasa Application in main. Asta e doar pentru claritatea exemplului.